test/test-scripts/helpers/drainUsersVoucher.js

```diff
@@ -0,0 +1,37 @@
+import { Contract, ZeroAddress } from "ethers";
+import {
+  VOUCHER_HUB_ADDRESS,
+  provider,
+  getVoucherManagerWallet,
+} from "./utils.js";
+import { voucherHubAbi } from "../abis/VoucherHub.js";
+
+export const drainUsersVoucher = async ({ ownerAddress, value = 1000 }) => {
```

Comment:
'value' parameter is not used 

```suggestion
export const drainUsersVoucher = async ({ ownerAddress}) => {
```